src/js/controller/model.js

```diff
@@ -264,11 +268,11 @@ define([
         this.loadVideo = function() {
             this.mediaController.trigger(events.JWPLAYER_MEDIA_PLAY_ATTEMPT);
             var idx = this.get('item');
-            this.getVideo().load(this.get('playlist')[idx]);
+            _provider.load(this.get('playlist')[idx]);
         };
 
         this.playVideo = function() {
-            this.getVideo().play();
+            _provider.play();
```

Comment:
```this.getVideo()``` is better because it abstracts away the fact that we have the current provider bound in the Model closure. I think this is important because we want to remove the _provider var from model at some point, and instead pass the commands to the mediaController